observe_configure_mac_script.sh

```diff
@@ -311,6 +319,9 @@ includeFilefluentAgent(){
               sudo cp "$config_file_directory/observe-mac-host.conf" /etc/fluent-bit/observe-mac-host.conf
               local daemon_directory="/Library/LaunchDaemons"
               [ -d "$daemon_directory" ] || sudo mkdir "$daemon_directory"
+              if [ $BASE_BREW = "/usr/local" ]; then
+                sed -i '' 's|/opt/homebrew|/usr/local|g' $config_file_directory/fluent-bit.plist
```

Comment:
don't you need to put `LC_ALL=C` in front of this sed to keep Mac from complaining? see other sed's in this script